ansible/roles/vm_set/library/vm_topology.py

```diff
@@ -194,15 +199,8 @@ def init(self, vm_set_name, topo, vm_base, duts_fp_ports, duts_name, ptf_exists=
 
         self.injected_fp_ports = self.extract_vm_vlans()
 
-        if ptf_exists:
-            self.pid = VMTopology.get_pid(PTF_NAME_TEMPLATE % vm_set_name)
-        else:
-            self.pid = None
-
         self.bp_bridge = ROOT_BACK_BR_TEMPLATE % self.vm_set_name
 
-        self.update()
```

Comment:
why move these blocks up?

Reply:
> why move these blocks up?

https://github.com/Azure/sonic-mgmt/blob/master/ansible/roles/vm_set/library/vm_topology.py#L184
get_bridge() will need self.host_ifaces, therefore it have to get host_ifaces before get_bridge()